docs/readthedocs/protocols.md

```diff
@@ -0,0 +1,30 @@
+(protocols)=
+### Initialization phase
+
+![Initialization Phase Sequence Diagram](http://www.plantuml.com/plantuml/proxy?cache=no&src=https://raw.githubusercontent.com/ie3-institute/simona/dev/docs/uml/protocol/InitializationPhase.puml)
+
+(protocolsTriggerParticipantAgent)=
+### Triggering of participant agents
+
+
+#### Participant triggered by primary data
+
+This activation protocol is used, if the agent is foreseen to only play back externally given, precalculated model
+results.
```

Comment:
This activation protocol is used, if the agent receives and is required to playback externally given, precalculated model
results.

Reply:
Thanks for your review, I included all of your feedback.

Reply:
I'd leave out the comma after `used`, but then again we've had enough review rounds right here.